index.js

```diff
@@ -0,0 +1,47 @@
+const express = require('express');
+const cors = require('cors');
+const helmet = require('helmet');
+
+const projectDb = require('./data/helpers/projectDb');
+const actionDb = require('./data/helpers/actionDb');
+
+const server = express();
+const port = 8500;
+
+server.use(cors());
+server.use(helmet());
+server.use(express.json());
+
+server.post('/projects', (req, res) => {
+    const project = req.body;
```

Comment:
Instead of taking the entire request body, specify which properties you want. This will stop malicious attacks to your database.